meta-lmp-base/recipes-support/lmp-auto-hostname/lmp-auto-hostname/lmp-auto-hostname.service.in

```diff
@@ -3,10 +3,12 @@ Description=Linux microPlatform Auto Hostname Update
 DefaultDependencies=no
 Before=network-pre.target avahi-daemon.service
 After=local-fs.target dbus.service
+ConditionPathExists=!/etc/lmp-update-hostname.sh.done
 
 [Service]
 Environment=MACHINE=@@LMP_HOSTNAME_MACHINE@@ MODE=@@LMP_HOSTNAME_MODE@@ NETDEVICE=@@LMP_HOSTNAME_NETDEVICE@@
 ExecStart=/usr/bin/lmp-update-hostname
+ExecStartPost=touch /etc/lmp-update-hostname.sh.done
```

Comment:
Having a stamp as a condition is ok, but one other option that could be done as well is simply disabling the service on ExecStartPost, kind of similar as we do with the resize-helper:

```
cat meta-lmp-base/recipes-support/resize-helper/resize-helper/resize-helper.service
[Unit]
Description=Resize root filesystem to fit available disk space
Wants=systemd-udevd.service systemd-udev-trigger.service
After=systemd-remount-fs.service systemd-udevd.service

[Service]
Type=oneshot
ExecStartPre=-/bin/udevadm settle
ExecStart=/usr/sbin/resize-helper
ExecStartPost=/bin/systemctl disable resize-helper.service

[Install]
WantedBy=basic.target
```

The advantage of disabling the service is that the user can force a manual start/restart when needed, without having to understand that a custom stamp file was used here.

Reply:
Thanks for the remark, Ricardo, it is a nice idea to use the same mechanism across the lmp.
I'll rework my branch to implement your proposal.

Reply:
Reworked with the remark.